common/beacon/client.go

```diff
@@ -19,7 +20,7 @@ func NewBeaconClient(ctx context.Context, cfg flags.BeaconConfig) (Client, error
 	cctx, cancel := context.WithCancel(ctx)
 	defer cancel()
 
-	c, err := http.New(cctx, http.WithAddress(cfg.BeaconURL), http.WithTimeout(cfg.BeaconClientTimeout))
+	c, err := http.New(cctx, http.WithAddress(cfg.BeaconURL), http.WithTimeout(cfg.BeaconClientTimeout), http.WithEnforceJSON(cfg.EnforceJSON), http.WithLogLevel(zerolog.ErrorLevel))
```

Comment:
why add WithLogLevel.ErrorLevel?

Reply:
Going to address in @0x00101010's comment